src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.ts

```diff
@@ -1,15 +1,95 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import {COMMA, ENTER, N} from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+export interface workType{
```

Comment:
lets put this in the types directory in a file called work-type.ts

Reply:
this is done